_gsocproposals/2019/proposal_ActsGpuRace.md

```diff
@@ -0,0 +1,72 @@
+---
+title: GPU Race for Particle Hunting
+layout: gsoc_proposal
+project: ACTS
```

Comment:
There is [another proposal under the Acts project](https://hepsoftwarefoundation.org/gsoc/2019/proposal_ActsRustKF.html) that uses lower case to name the project. Should we use the same on this case? Otherwise both proposals will be listed as part of two different projects once the webpage is generated.

Reply:
Sure, we want the same "Acts" everywhere.